src/gantry/widgets.py

```
@@ -45,6 +47,105 @@ def __init__(self, *args, **kwargs):
+            if not key_str.startswith("col_"):
+                continue
+            try:
+                i = int(key_str.split("_")[1])
+            except (ValueError, IndexError):
+                continue
+            if i >= len(self._columns):
+                continue
+            column.label = Text(self._build_column_label_text(self._columns[i], i))
+        self.refresh()
+
+    def _coerce_sort_value(self, value: Any) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)